docs/proposed/psa-driver-interface.md

```diff
@@ -586,6 +632,10 @@ How does a driver author decide which location values to use? It should be possi
 
 Can the driver assembly process generate distinct location values as needed? This can be convenient, but it's also risky: if you upgrade a device, you need the location values to be the same between builds.
 
+#### Multiple transparent drivers
+
+When multiple transparent drivers implement the same mechanism, which one is called? The first one? The last one? Unspecified? Or is this an error (excluding capabilities with fallback enabled)?
```

Comment:
I guess in theory a platform could have multiple accelerators with an intersecting set of mechanisms. The drivers for those accelerators may come from different sources and it could be a pain for the system integrator if this were considered an error. In the fallback case, you say "The core considers drivers in the order given by the driver description list" so I think we should do the same here too.

Reply:
Updated. I think this may not be fully settled, so I'm keeping the entry in the “open questions” section for now.